recipes/biodigest/meta.yaml

```diff
@@ -10,16 +10,21 @@ source:
   sha256: 7618cedf40fedd6dd60aa2fbf8635fb4af0a2c0ae16c01ef9741d30729b6e8bc
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation"
+  number: 1
+  script: 
+    - "{{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation"
   noarch: python
   run_exports:
     - {{ pin_subpackage('biodigest', max_pin="x.x") }}
 
+channels:
```

Comment:
This should not be needed

Reply:
Did you accidentally re-add this section?

Reply:
yeah, the graph-tool import in host fails otherwise in the pipeline, but I will try again after adding the test wether the build also succeeds when deleting the channel section.

Reply:
It definitely works for me and I’m 99% sure it will work on CI as well because conda-forge and bioconda are preconfigured in the channels list already.

Have you maybe not configured conda-forge and/or bioconda in your `~/.condarc` channels list?